### PR TITLE
updated build script to trigger the bucket creation after the build

### PR DESCRIPTION
**Pull request by PaNaVTEC**

### CODE REVIEW

File: _utility/deploy_prb.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cd _site
+
+WEBSITE_INDEX=index.html
+BUCKET_NAME=codurance-website-preview
+
+aws s3 mb s3://$BUCKET_NAME --region $AWS_REGION
```
Comment (cyplo) [inline _utility/deploy_prb.sh]: Would that make every pull request deploy to the same bucket ? Would that make different pull requests have same preview url ?

Comment (neomatrix369) [inline _utility/deploy_prb.sh]: We are working on creating dynamic buckets and also clean them up once they are not needed anymore.

So to answer to your question - different URLs from different buckets.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  try {
+
+    stage('Checkout') {
+      checkout scm
+    }
+
+    stage('Build') {
+      sh '''#!/bin/bash -l
+        source ~/.bashrc
```
Comment (PaNaVTEC) [inline Jenkinsfile]: It's a machine-specific file, used to load Ruby into the PATH. As the plan is to migrate to a hosted CI service, we won't need a custom Ruby installation anyway, so not commiting the contents of this file now - treating it as something needed on the hostfor the script to function.
